Sources/Runestone/TextView/Core/TextView.swift

```diff
@@ -568,6 +568,8 @@ open class TextView: UIScrollView {
             textInputView.lineEndings = newValue
         }
     }
+    /// If this flag is enable the TextView will present the edit menu when select a range 
```

Comment:
Can we change this to be slightly more descriptive?

```suggestion
    /// When enabled the text view will present a menu with actions actions such as Copy and Replace after navigating to a highlighted range.
```